### PR TITLE
Add printAthenaCreateTable() support

### PR DESCRIPTION
**Pull request by nsphung**
Add printAthenaCreateTable() support #23

It's about the same as in spark-daria.

### CODE REVIEW

File: quinn/dataframe_helpers.py [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
 def two_columns_to_dictionary(df, key_col_name, value_col_name):
     k, v = key_col_name, value_col_name
     return {x[k]: x[v] for x in df.select(k, v).collect()}
 
+
 def to_list_of_dictionaries(df):
     return list(map(lambda r: r.asDict(), df.collect()))
+
+
+def print_athena_create_table(df, athena_table_name, s3location):
```
Comment (afranzi) [inline quinn/dataframe_helpers.py:14]: shouldn't be easier to return a `str` and then allow the developer to print it or use it with any athena connector?

Comment (nsphung) [reply quinn/dataframe_helpers.py:14]: I can but I was trying to stay connected with the Scala implementation in spark-daria [def printAthenaCreateTable(df: DataFrame, athenaTableName: String, s3location: String): Unit](https://github.com/MrPowers/spark-daria/blob/93cb343bd8c11c241ed3487e2c77961f2d63f020/src/main/scala/com/github/mrpowers/spark/daria/sql/DataFrameHelpers.scala#L193)

Should I change it ? If so, what kind of result/format string you expect ?

Comment (afranzi) [reply quinn/dataframe_helpers.py:14]: Maybe we could have the `athena_create_table` that returns the `str` and then we keep the `print_athena_create_table` to stay connected with the scala implementation?

Comment (MrPowers) [reply quinn/dataframe_helpers.py:14]: @afranzi - I like that idea.  I'll merge this in, but think the `athena_create_table` would be a good addition too!

**Comment by MrPowers** [comment]
@nsphung - thanks for the contribution!  Keep them coming ;)

**Pull request merged**